components/crud-web-apps/README.md

```diff
@@ -0,0 +1,8 @@
+# Build instructions for docker images
+To build the Jupyter and Tensorboards docker images directly (without the Makefile) for development purposes, the following commands can be used from this directory.
+
+## Building the Jupyter Images
+`cp jupyter/Dockerfile.dockerignore .dockerignore && docker build --pull -f "jupyter/Dockerfile" -t <image-tag> . && rm .dockerignore`
```

Comment:
I think it's difficult to read one big command so could you instead use the triple \` markdown and break the commands with `\`? It would look like this
```bash
cp jupyter/Dockerfile.dockerignore \
&& docker build --pull -f "jupyter/Dockerfile" -t ${IMG} \
&& rm .dockerignore
```